azure-pipelines.yml

```diff
@@ -32,6 +32,10 @@ jobs:
           yarn checkchange
         displayName: check change
 
+      - script: |
+          yarn nx workspace-lint
+        displayName: nx:workspace-lint
```

Comment:
run nx workspace lint checks on ci to prevent getting out of sync whilst doing changes by hand